ietf/status/models.py

```diff
@@ -0,0 +1,22 @@
+# Copyright The IETF Trust 2017-2020, All Rights Reserved
```

Comment:
this is a new model copyright should be 2024 - same with all other new files. Modified files should have the date range extended to 2024